test/integration/targets/inventory/runme.sh

```diff
@@ -112,3 +112,6 @@ ansible-inventory  -i inv_with_int.yml --list "$@"
 	cd 1/2
 	ansible-playbook -e @../vars.yml -i 'web_host.example.com,' -i inventory.yml 3/extra_vars_relative.yml "$@"
 }
```

Comment:
the last test cds into a subdir so the relative vault password file isn't found
```suggestion
	cd ../..
}
```